oak/server/rust/oak_runtime/src/node/grpc/server.rs

```diff
@@ -378,6 +382,16 @@ impl GrpcRequestHandler {
                         m.data
                     })
                 })
+                .and_then(|response| {
+                    // Decode [`prost_types::Any`] message.
+                    GrpcResponse::decode(response.as_slice())
+                        .map_err(|error| {
+                            error!("Couldn't parse a GrpcResponse message: {}", error);
```

Comment:
I'm still confused about the "a"s in the log messages, but I guess I'll get used to it :)

Reply:
Changed to `the` (hope it's right)